src/CustomItem.cs

```diff
@@ -65,13 +59,31 @@ public CustomItem(
 
 		Main.Log($"Loaded prefabs for {itemInfo.Name}");
 
+		var previewBounds = itemInfo.PreviewBounds;
+		Main.Log($"Preview bounds: {previewBounds}");
+		if (previewBounds == default)
+		{
+			var collider = ProvidedPrefab.GetComponentInChildren<BoxCollider>();
+			if (collider != null)
+			{
+				previewBounds = collider.size;
+			}
+			else
+			{
+				previewBounds = new Vector3(0.2f, 0.2f, 0.2f);
+			}
+		}
```

Comment:
I propose to change this to:
```
var previewBounds = itemInfo.PreviewBounds;
if (previewBounds == default)
{
	Main.Warning($"{Name} has no preview bounds, setting defaults");
	previewBounds = new Vector3(0.5f, 0.5f, 0.5f);
}
Main.Log($"Preview bounds: {previewBounds}");
```

Grabbing a random BoxCollider doesn't work as intended because it just grabbed a collider from Blahaj's leg I think. Anyway I think it's better to just use a default size if itemInfo.PreviewBounds isn't set.

The default size of 0.2 was too small for my items, so let's increase it to 0.5.

Reply:
Also I just realized I renamed this field in the next branch...

Reply:
Alright, so I've removed the box collider approach.

The 0.2 being too small was a challenge, but 0.5 resulted in my test items not being able to be added to most shops.  I played around with it and set it for 0.35 width, 0.3 height for now - I think that should work for _most_ regular items